tests/providers/test_online.py

```diff
@@ -0,0 +1,31 @@
+# Test for one implementation of the interface
+from lexicon.providers.online import Provider
+from integration_tests import IntegrationTests
+from unittest import TestCase
+import pytest
+
+# Hook into testing framework by inheriting unittest.TestCase and reuse
+# the tests which *each and every* implementation of the interface must
+# pass, by inheritance from integration_tests.IntegrationTests
+class OnlineProviderTests(TestCase, IntegrationTests):
+
+	Provider = Provider
+	provider_name = 'online'
+	domain = 'edasarl.fr'
+
+	def _filter_headers(self):
+		return ['Authorization']
+
+	def _test_engine_overrides(self):
+		overrides = super(OnlineProviderTests, self)._test_engine_overrides()
+		overrides['fallbackFn'] = (lambda x: 'placeholder_' + x if x != 'priority' else '')
+		return overrides
+
+	@pytest.fixture(autouse=True)
+	def skip_suite(self, request):
```

Comment:
Hey, This PR looks just about perfect but it seems you're skipping the extended suite 
Can you remove lines 24 - 27 and ensure that your provider supports record sets?

Reply:
Not that i don't want to, but i'm confused about the meaning of record sets.
In this provider, one can only modify a *disabled* record set, and need to activate it to make it live.
This means, in particular, that the id of a given record will change when the record set it is being part of is made active.
Is this compatible with how lexicon deal with things ?

Reply:
@lexicon maybe i replied that fast that you did not notice :)

Reply:
Looking at the documentation, it looks like only zones can be enabled/disabled, not record sets. 

fundamentally the request is that you can have multiple records for same name & type, with differing content.




Type | Name | Content
------------ | ------------- | -------------
A | test1.example.com | 1.2.3.4 
A | test2.example.com | 4.3.2.1
TXT | _acme-challenge.example.com | D-52Wm4V7xoUpGax-F8FrPO45cQRcbRj-XoblaY4uYM
TXT | _acme-challenge.example.com | 6lOgCI0p_LRhtrJMh9aTYAek6hZ64nT75-DkeeQccfA

Reply:
oh right, indeed if record set is only that feature, it's easy to support.